backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5756150_sys_Add_Parameter_To_BusinessRule_Messages.sql

```diff
@@ -0,0 +1,23 @@
+
+SELECT backup_table('AD_Message')
+;
+
+SELECT backup_table('AD_Message_TRL')
+;
+
+UPDATE AD_Message m
+SET MsgText = CONCAT(m.MsgText, ' {}')
```

Comment:
I'm fine with using CONCAT, but it looks a bit awkward. I'm wondering why you didn't use `m.MsgText || ' {}'`...